recipes/mne-python/construct.yaml

```diff
@@ -112,9 +113,8 @@ specs:
   - xlrd =2.0.1
   # Statistics
   - pingouin =0.5.3
-  - pycircstat =0.0.2
```

Comment:
This appears to be unmaintained, relies on `nose` for example and meaningful commits are from a while ago

https://github.com/circstat/pycircstat

It creates a conflict, so let's remove it https://github.com/mne-tools/mne-installers/actions/runs/6871571108/job/18688630912?pr=225

Reply:
Ok!